proxy/server.py

```diff
@@ -596,7 +597,10 @@ def do_GET(self):
                 'soe': pw.level() or 0,
                 'grid_status': int(pw.grid_status() == 'UP'),
                 'reserve': pw.get_reserve() or 0,
-		'time_remaining_hours': pw.get_time_remaining() or 0
+                'time_remaining_hours': pw.get_time_remaining() or 0,
+                'full_pack_energy': get_value(d, 'nominal_full_pack_energy') or 0,
+                'energy_remaining': get_value(d, 'nominal_energy_remaining') or 0,
+                'strings': pw.strings(jsonformat=True) or {}
```

Comment:
We will want the pythonic dictionary values for strings instead of JSON, so we need to change this to `jsonformat=False`.